packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -736,4 +765,13 @@ contract ValidatorManager is ISemver, IValidatorManager {
             _validatorInfo[validator].noSubmissionCount = 0;
         }
     }
+
+    /**
+     * @notice Allow restrictedOutputIndex.
```

Comment:
```suggestion
     * @notice Sets the output index that only a trusted validator can submit.
               This can only be called by a Trusted Validator.
```